vendor/engines/secure_rooms/app/models/secure_rooms/ability_extension.rb

```diff
@@ -0,0 +1,17 @@
+module SecureRooms
+
+  class AbilityExtension
```

Comment:
I was wondering if we could forego this ability extension and rely on the `Product` permissions, but that probably doesn't make sense. This probably gives us better control. We can always change it later.